.vscode/extensions.json

```diff
@@ -8,6 +8,7 @@
         "rvest.vs-code-prettier-eslint",
         "silvenon.mdx",
         "ms-dotnettools.csharp",
-        "ms-dotnettools.blazorwasm-companion"
+        "ms-dotnettools.blazorwasm-companion",
+        "lucono.karma-test-explorer"
```

Comment:
Being able to debug in VSCode instead of an external browser is a HUGE improvement! I think it'd be worth calling this out in CONTRIBUTING with a screenshot or a brief note. I don't see an existing section for it so maybe a new section called "Running Tests" in between "Coding Conventions" and "Pull request policies"?